src/storage/mod.rs

```diff
@@ -990,15 +992,30 @@ impl<E: Engine, L: LockMgr> Storage<E, L> {
                                 !ctx.get_not_fill_cache(),
                             );
                             let mut results = vec![];
-                            // TODO: optimize using seek.
+                            let use_incremental_get = gets.len() >= BATCH_INCREMENTAL_GET_LIMIT;
+                            if use_incremental_get {
```

Comment:
Incremental get can be always used (it supports unordered get now). Sort can be optionally performed.